data/core/units/boats/Galley.cfg

```diff
@@ -0,0 +1,137 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Galley
+    name= _ "Galley"
+    race=transport
+    image="units/transport/galley/galley.png"
+    # ideally, we have "crew" that make sense for dialogs, and "mechanical" that depicts the transport
+    # but for now, we just have "crew"
+    small_profile="portraits/transport/mechanical-galley.webp"
+    profile="portraits/transport/crew-boat.webp"
+    hitpoints=36
+    movement_type=ship
+    [movement_costs]
+        deep_water=2
+        village=1
+    [/movement_costs]
+    [defense]
+        deep_water=50
+        shallow_water=40
+        village=50
+    [/defense]
+    movement=5
+    experience=50
+    level=1
+    alignment=lawful
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=15
+    usage=null
+    description= _"While Galleons and Cogs rely on impressive sails to harness the power of the wind, Galleys are propelled by muscle-power of the crew manning the oars.  This has the disadvantage of requiring a larger crew, and it does limit the speed and efficiency of transport.  But these seeming weaknesses also make the vessel less vulnerable to the whims of weather and the predations of pirates."
```

Comment:
Is it okay for you to only use one space after the full-stop? Often we have to correct this non-standard spacing after a merge.

Reply:
Why do you have to correct it?

Reply:
Because it's a string marked for translation and in poedit, at least, non-standard spacing like this is often marked as an issue.

Reply:
It's not "non-standard"...  Can you point to a specific instance of trouble?  Maybe we can help fix poedit.

Reply:
Well, I'm not the only one who's done it, but for consistency in Wesnoth we usually avoid the double-spacing. If you don't want to change it, then fine.

Reply:
> It's not "non-standard"... Can you point to a specific instance of trouble? Maybe we can help fix poedit.

It's non-standard in wesnoth since many years.